### DISCUSSION TITLE
Setting Align text for TextInput and Placeholder

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#4034] [Help] [answered]
Hi All,

How to setting individual alignment for TextInput and Placeholder ?

I want to set alignment to the right or center for some TextInput and Placeholder.

Thank You

**Comment by jigujigu** [comment, marked as answer]
Solved,
with extraInputAttributes. 

```
TextInput::make('string_field')
     ->extraInputAttributes(['style' => 'text-align:right']),
```

**Reply by albertlens** [reply]
I am trying to use the same, both with extraAttributes and extraInputAttributes and no result seen. Maybe I have to run some npm command to see changes?

I want a certain TextInput to be a bigger font size and be bolder.

Thanks.
